prometheus/promhttp/http.go

```diff
@@ -10,22 +10,20 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-// Copyright (c) 2013, The Prometheus Authors
-// All rights reserved.
 //
-// Use of this source code is governed by a BSD-style license that can be found
-// in the LICENSE file.
-
-// Package promhttp contains functions to create http.Handler instances to
-// expose Prometheus metrics via HTTP. In later versions of this package, it
-// will also contain tooling to instrument instances of http.Handler and
-// http.RoundTripper.
+// Package promhttp provides tooling around HTTP servers and clients.
```

Comment:
There has to be a true empty line before this line to make golint happy and make godoc recognize the package doc comment.